src/yki/boundary/cas.clj

```diff
@@ -30,14 +34,40 @@
                                      :headers      {"Caller-Id" caller-id}})))
 
 (defn- json-request [^String method url data]
-  (let [json-str        (json/write-str data)
+  (let [body            (when data (json/write-str data))
         request-builder (RequestBuilder. method)]
     (doto request-builder
       (.addHeader "Content-Type" "application/json")
-      (.setBody json-str)
+      (.setBody body)
       (.setUrl url))
     (.build request-builder)))
 
+(defn- clear-ticket-stores! [^CasClient cas-client]
+  (let [cls                   (.getClass cas-client)
+        session-fetcher-field (.getDeclaredField cls "casSessionFetcher")
+        _                     (.setAccessible session-fetcher-field true)
+        session-fetcher       (.get session-fetcher-field cas-client)]
+    (.clearTgtStore ^CasSessionFetcher session-fetcher)
+    (.clearSessionStore ^CasSessionFetcher session-fetcher)))
```

Comment:
Hieman nasty reflektiokikkailu, mutta oleellisesti tuohon `CasClientImpl`-luokan `private final CasSessionFetcher casSessionFetcher` fieldiin tarttee päästä käsiksi.

Toinen, mahdollisesti siistimpi vaihtoehto olisi tietty konstruoida itse tarvittava `CasSessionFetcher`-instanssi ja antaa se parametriksi `CasClientImpl`-konstruktorille. Tällöin reflektiota ei tarvita, mutta silloin pitää taas tietää hieman enemmän tuon java-cas-toteutuksen muista osista. Silloin pitäisi käytännössä itse tehdä ne temput mitä `CasClientBuilder`in `build(CasConfig config)` tarjoaa valmiina.